src/query/vacuum.c

```diff
@@ -1092,6 +1092,63 @@ xvacuum (THREAD_ENTRY * thread_p)
 #endif /* SA_MODE */
 }
 
+/*
+ * xvacuum_dump - Dump the contents of vacuum
+ *
+ * return: nothing
+ *
+ *   outfp(in): FILE stream where to dump the vacuum. If NULL is given,
+ *            it is dumped to stdout.
+ */
+void
+xvacuum_dump (THREAD_ENTRY * thread_p, FILE * outfp)
+{
+  LOG_PAGEID min_log_pageid = NULL_PAGEID;
+  int archive_number;
+
+  if (outfp == NULL)
+    {
+      outfp = stdout;
+    }
+
+  /* vacuum_boot () != NO_ERROR || prm_get_bool_value (PRM_ID_DISABLE_VACUUM) == true */
+  if (!vacuum_is_safe_to_remove_archives ())
+    {
+      return;
+    }
+
+  min_log_pageid = vacuum_min_log_pageid_to_keep (thread_p);
+  if (min_log_pageid == NULL_PAGEID)
+    {
+      /* this is an assertion case but ignore. */
```

Comment:
If it don't be returned a message, I think, user will get empty message.
You need to return warning message.

Reply:
This cannot also occur in the case of executing the vacuum_boot() successfully. Therefore, I will display the message 'vacuum did not boot properly' as a result like the previous case.

Reply:
OK.